classes/native/src/lib.rs

```diff
@@ -1,7 +1,5 @@
-#[macro_use]
```

Comment:
Not really relevant to this line, but I don't have a good place to the comment. Do you think `macro_use` should be removed from _all_ of the examples and swapped for `use` statements on the macros?